### PR TITLE
Use tchannel 0.24

### PR DESCRIPTION
**Pull request by breerly**
@yarpc/python

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Releases
 0.6.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Pin to ``tchannel>=0.24,<0.24``.
```
Comment (abhinav) [inline CHANGELOG.rst]: `< 0.25`

Comment (breerly) [inline CHANGELOG.rst]: blarg

**Comment by abhinav** [comment]
yay

**Comment by blampe** [comment]
Should we start a yarpc branch to develop against?

**Comment by breerly** [comment]
@blampe i think we're getting pretty close to 1.0, so in that case, I'd say probably not

**Pull request merged**